### PR TITLE
final + micro optimizations

### PR DESCRIPTION
**Pull request by pizzi80**

### CODE REVIEW

File: src/main/java/org/omnifaces/exceptionhandler/FullAjaxExceptionHandler.java [hunk 1]
```diff
@@ -310,15 +310,15 @@ public String getMessage() {
     private static final String ERROR_PAGE_ERROR =
         "<?xml version='1.0' encoding='UTF-8'?><partial-response id='error'><changes><update id='jakarta.faces.ViewRoot'>"
             + "<![CDATA[<html lang='en'><head><title>Error in error</title></head><body><section><h2>Oops!</h2>"
-            + "<p>A problem occurred during processing the ajax request. Subsequently, another problem occurred during"
-            + " processing the error page which should inform you about that problem.</p><p>If you are the responsible"
-            + " web developer, it's time to read the server logs about the bug in the error page itself.</p></section>"
+                + "<p>A problem occurred during processing the ajax request. Subsequently, another problem occurred during"
+                + " processing the error page which should inform you about that problem.</p><p>If you are the responsible"
+                + " web developer, it's time to read the server logs about the bug in the error page itself.</p></section>"
```
Comment (BalusC) [inline src/main/java/org/omnifaces/exceptionhandler/FullAjaxExceptionHandler.java]: Why?

Comment (pizzi80) [reply src/main/java/org/omnifaces/exceptionhandler/FullAjaxExceptionHandler.java]: don't know, probably an IDE error? ... I'll check!

File: src/main/java/org/omnifaces/exceptionhandler/FullAjaxExceptionHandler.java [hunk 2]
```diff
@@ -309,16 +309,16 @@ public String getMessage() {
             + " Trying to render a hardcoded error page now.";
     private static final String ERROR_PAGE_ERROR =
         "<?xml version='1.0' encoding='UTF-8'?><partial-response id='error'><changes><update id='jakarta.faces.ViewRoot'>"
-            + "<![CDATA[<html lang='en'><head><title>Error in error</title></head><body><section><h2>Oops!</h2>"
+        + "<![CDATA[<html lang='en'><head><title>Error in error</title></head><body><section><h2>Oops!</h2>"
             + "<p>A problem occurred during processing the ajax request. Subsequently, another problem occurred during"
             + " processing the error page which should inform you about that problem.</p><p>If you are the responsible"
             + " web developer, it's time to read the server logs about the bug in the error page itself.</p></section>"
-            + "</body></html>]]></update></changes></partial-response>";
+        + "</body></html>]]></update></changes></partial-response>";
```
Comment (BalusC) [inline src/main/java/org/omnifaces/exceptionhandler/FullAjaxExceptionHandler.java]: Please reset the changes.

**Comment by BalusC** [comment]
LGTM but there's merge conflict on ImportFunctions. Please upmerge the current branch (you added a METHOD_PARAM_COUNT_COMPARATOR there).

**Pull request merged**